src/metabase/analytics/stats.clj

```diff
@@ -126,31 +126,34 @@
+   :running_on                           (environment-type)
+   :startup_time_millis                  (public-settings/startup-time-millis)
+   :application_database                 (config/config-str :mb-db-type)
+   :check_for_updates                    (public-settings/check-for-updates)
+   :report_timezone                      (driver/report-timezone)
    ; We deprecated advanced humanization but have this here anyways
-   :friendly_names                      (= (humanization/humanization-strategy) "advanced")
-   :email_configured                    (email/email-configured?)
-   :slack_configured                    (slack/slack-configured?)
-   :sso_configured                      (google/google-auth-enabled)
-   :instance_started                    (snowplow/instance-creation)
-   :has_sample_data                     (t2/exists? Database, :is_sample true)
-   :enable_embedding                    (embed.settings/enable-embedding)
-   :embedding_app_origin_set            (boolean (embed.settings/embedding-app-origin))
-   :appearance_site_name                (not= (public-settings/site-name) "Metabase")
-   :appearance_help_link                (public-settings/help-link)
-   :appearance_logo                     (not= (public-settings/application-logo-url) "app/assets/img/logo.svg")
-   :appareance_favicon                  (not= (public-settings/application-favicon-url) "app/assets/img/favicon.ico")
-   :apperance_loading_message           (not= (public-settings/loading-message) :doing-science)
-   :appearance_metabot_greeting         (not (public-settings/show-metabot))
-   :apparerance_lighthouse_illustration (not (public-settings/show-lighthouse-illustration))
-   :appearance_ui_colors                (appearance-ui-colors-changed?)
-   :appearance_chart_colors             (appearance-chart-colors-changed?)
-   :appearance_show_mb_links            (not (public-settings/show-metabase-links))})
+   :friendly_names                       (= (humanization/humanization-strategy) "advanced")
+   :email_configured                     (email/email-configured?)
+   :slack_configured                     (slack/slack-configured?)
+   :sso_configured                       (google/google-auth-enabled)
+   :instance_started                     (snowplow/instance-creation)
+   :has_sample_data                      (t2/exists? Database, :is_sample true)
+   :enable_embedding                     (embed.settings/enable-embedding)
+   :embedding_app_origin_set             (boolean (embed.settings/embedding-app-origin))
+   :appearance_site_name                 (not= (public-settings/site-name) "Metabase")
+   :appearance_help_link                 (public-settings/help-link)
+   :appearance_logo                      (not= (public-settings/application-logo-url) "app/assets/img/logo.svg")
+   :appearance_favicon                   (not= (public-settings/application-favicon-url) "app/assets/img/favicon.ico")
```

Comment:
Use hide whitespace to review this file to see the actual diff
![image](https://github.com/metabase/metabase/assets/1937582/d9d5c7da-434d-4e91-9ffc-697293c5ba93)